src/libraries/System.Net.Quic/src/Interop/Interop.MsQuic.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Net.Quic.Implementations.MsQuic.Internal;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static  class MsQuic
+    {
+#pragma warning disable BCL0015 // Invalid Pinvoke call
```

Comment:
Just curious, what makes this "Invalid"?

Reply:
Issue in the repo: pinvoke is fine but there are many analyzers that think this is invalid.